upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -1433,8 +1433,12 @@ func initializeOpenstack(opt *NewClusterOptions, cluster *api.Cluster) {
 }
 
 func createEtcdCluster(etcdCluster string, controlPlanes []*api.InstanceGroup, encryptEtcdStorage bool, etcdStorageType string) api.EtcdClusterSpec {
-	etcd := api.EtcdClusterSpec{}
-	etcd.Name = etcdCluster
+	etcd := api.EtcdClusterSpec{
+		Name: etcdCluster,
+		Manager: &api.EtcdManagerSpec{
+			BackupRetentionDays: fi.PtrTo[uint32](90),
```

Comment:
Why explicitly lock this down in new clusters?

Reply:
1. don't want to change behaviour for those that expect it to be longer
2. want it explicit to make it easy to change by operators

Reply:
Right - the idea (AIUI) is to populate the value explicitly (only) for new clusters created via `kops create cluster`, so users see the value and can easily change it.